Common/TableProducer/PID/pidTPC.cxx

```diff
@@ -68,18 +68,20 @@ struct tpcPid {
   o2::pid::tpc::Response* responseptr = nullptr;
   // Network correction for TPC PID response
   Network network;
+  std::string networkPathOnAlien = ""; // Will be set by the user or automatically if TPCPIDResponse object is fetched from CCDB
```

Comment:
Maybe you can consider using a configurable and having this set to override the choice via CCDB e.g. CCDB is used only if this is left empty

Reply:
Indeed this is exactly what is being done. This is just a global variable which is changed according to the configurable networkPathAlien. If the configurable is left empty, the objects are automatically fetched using the runnumber from the TPC PID response object (metadata). If this configurable is set however, the given path on AliEn is used to fetch the network file.